web/packages/teleport/src/Integrations/Enroll/AwsConsole/IamIntegration/IamIntegration.tsx

```diff
@@ -0,0 +1,434 @@
+  }
+
+  return (
+    <>
+      <Flex mt={3} justifyContent="space-between" alignItems="center">
+        <H2>AWS CLI / Console Access</H2>
+        <InfoGuideButton
+          config={{
+            guide: <Guide resourcesRoute={resourceRoute} />,
+          }}
+        />
+      </Flex>
+      <Box>
+        Compatible with any CLI and AWS SDK-based tootling (includes Terraform,
+        AWS CLI). Teleport uses <b>AWS IAM Roles Anywhere</b> to manage access
+        and allows you to configure the right permissions for your users.
+        <br />
+        Follow the below steps to create a Roles Anywhere Trust Anchor and
+        configure the required IAM Roles for synchronizing Profiles as Teleport
+        resources.
+      </Box>
+      <Alert
+        kind="neutral"
+        icon={Info}
+        mt={5}
+        secondaryAction={manageAction}
+        details={
+          'Create AWS Profiles and assign Roles to them in your AWS account. Teleport will allow you to import these Profiles as Resources.'
+        }
+      >
+        Prerequisites
+      </Alert>
+      <Validation>
+        {({ validator }) => (
+          <Flex flexDirection="column" gap={3}>
+            <CardTile>
+              <Flex flexDirection="column">
+                <H2>Step 1: Name your Teleport Integration</H2>
+                <Subtitle2>Give this integration a name.</Subtitle2>
+              </Flex>
+              <Flex flexDirection="column" gap={1} maxWidth={500}>
+                <FieldInput
+                  readonly={phase !== Phase.One}
+                  label="Integration Name*"
+                  placeholder="teleport-aws-prod"
+                  rule={requiredField('Name is required')}
+                  value={formState.integrationName}
+                  onChange={(e: ChangeEvent<HTMLInputElement>) => {
+                    return setFormState(prev => ({
+                      ...prev,
+                      integrationName: e.target.value,
+                    }));
+                  }}
+                />
+              </Flex>
+              <CollapsibleInfoSection
+                openLabel="Optionally edit Trust Anchor, IAM Role and Profile Names"
+                closeLabel="Edit Trust Anchor, IAM Role and Profile Names"
+              >
+                <Flex flexDirection="column" gap={1} maxWidth={500}>
+                  <FieldInput
+                    readonly={phase !== Phase.One}
+                    label="Trust Anchor Name"
+                    placeholder=""
+                    value={formState.trustAnchorName}
+                    onChange={(e: ChangeEvent<HTMLInputElement>) => {
+                      return setFormState(prev => ({
+                        ...prev,
+                        trustAnchorName: e.target.value,
+                      }));
+                    }}
+                  />
+                  <FieldInput
+                    readonly={phase !== Phase.One}
+                    label="AWS IAM Role Name *"
```

Comment:
does the asteriks mean required? if so, i think the way we do it in other places is just let the validator handle the required field, and the optional field should be explicitly stated like `(optional)` or `- optional`